server/migrations/20241105200700-update-lixee-tic-features.js

```diff
@@ -0,0 +1,893 @@
+const Promise = require('bluebird');
+const db = require('../models');
+const logger = require('../utils/logger');
+const { DEVICE_FEATURE_CATEGORIES, DEVICE_FEATURE_TYPES } = require('../utils/constants');
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    // Get Zigbee2mqtt service
+    const service = await db.Service.findOne({
+      where: {
+        name: 'zigbee2mqtt',
+      },
+    });
+    if (service === null) {
+      logger.info('Zigbee2mqtt service not found.');
+      return;
+    }
+    logger.info(`Zigbee2mqtt migration: Found service zigbee2mqtt = ${service.id}`);
+    const zigbee2mqttDevices = await db.Device.findAll({
+      where: {
+        service_id: service.id,
+      },
+    });
+    logger.info(`Zigbee2mqtt migration: Found ${zigbee2mqttDevices.length} zigbee2mqtt devices`);
+
+    const lixeeTicdevice = await db.Device.findOne({
```

Comment:
What if the user have multiple Lixee TIC?

Reply:
Since I only have one Linke at home, I considered it as a general rule, but you're right, we can have multiple devices. I'll modify it.

Reply:
Nice! Did you re-test the migration since then? As there is no unit-tests on migration, it's very important to test them!

Reply:
Yes I created a second Lixee TIC in my db to test it. But it would be better if someone could also test it in real. I’m going to add a test image on the forum

Reply:
No worries, if you tested yourself it's good for me!